src/odemis/driver/tmcm.py

```diff
@@ -2647,6 +2647,12 @@ def __init__(self, name, role, channel, node_id, datasheet, axes, ustepsize,
         if not (refproc == REFPROC_STD or refproc is None):
             raise ValueError("Reference procedure %s unknown" % (refproc,))
 
+        # Get path of datasheet
+        if datasheet[0] != "/":
+            # it's a relative path
+            filedir = os.path.dirname(__file__)
+            datasheet = os.path.join(filedir, datasheet)
```

Comment:
I'd suggest the same code as we use already in the andorcam2 (and other drivers):
```python
if not os.path.isabs(datasheet):
    datasheet = os.path.join(os.path.dirname(__file__), datasheet)
```

Reply:
That's indeed cleaner. I didn't know about the isabs function, very useful!
I pushed the change.